addons/t9n/models/project.py

```diff
@@ -8,6 +8,7 @@ class Project(models.Model):
     _name = "t9n.project"
     _description = "Translation project"
 
+    name = fields.Char(required=True)
```

Comment:
same string suggestion so it's prettier/clearer in the views
```suggestion
    name = fields.Char('Project', required=True)
```